pkg/validation/internal/operatorhub.go

```diff
@@ -118,7 +118,7 @@ import (
 // `k8s-version` key is allowed. If informed, it will perform the checks against this specific Kubernetes version where the
 // operator bundle is intend to be used and will raise errors instead of warnings.
 // Currently, this check is capable of verifying the removed APIs only for Kubernetes 1.22 version.
-var OperatorHubValidator interfaces.Validator = interfaces.ValidatorFunc(validateOperatorHub)
+var OperatorHubValidator interfaces.Validator = interfaces.ValidatorFunc(validateOperatorHubDeprecated)
```

Comment:
Add a GoDoc deprecation comment as well.
```suggestion
//
// Deprecated: Use OperatorHubV2Validator and StandardCategoriesValidator for equivalent validation.
var OperatorHubValidator interfaces.Validator = interfaces.ValidatorFunc(validateOperatorHubDeprecated)
```